pipsi/__init__.py

```diff
@@ -53,7 +53,7 @@ def run(argv, **kw):
 
 def proc_output(s):
     s = s.strip()
-    if not isinstance(s, str):
```

Comment:
python2 aliases `bytes` to `str`, lets check for `isinstance(s, bytes)`

Reply:
I never knew that, thanks! I've updated the PR with this change